tests/e2e/helpers/keys.ts

```diff
@@ -193,6 +194,39 @@ export async function populateSetWithMembers(host: string, port: string, keyArgu
     });
 }
 
+/**
+ * Populate Zset key with members
+ * @param host The host of database
+ * @param port The port of database
+ * @param keyArguments The arguments of key and its members
+ */
+ export async function populateZSetWithMembers(host: string, port: string, keyArguments: AddKeyArguments): Promise<void> {
```

Comment:
Please set `number` on `port` instead of using the `Number` bellow

Reply:
I use port as a string in this method because we are storing port as string in conf file for all database types:
<img width="817" alt="image" src="https://user-images.githubusercontent.com/90257586/205017089-446b250b-e266-4f10-9dd6-f05429490da3.png">

so if I use port as number here then Number should be used in arguments everywhere

Reply:
yes, because we add data from this config as arguments for such methods as ``populateZSetWithMembers``